docs/styles.css

```diff
@@ -0,0 +1,9 @@
+/* Extra styles for the Nexus Mods App docs */
+
+.md-tabs {
+    background-color: var(--nexus-background-tertiary);
+}
+
+[dir=ltr] .md-header__title {
+    margin-left: 0
+}
```

Comment:
This should probably be added to https://github.com/Nexus-Mods/NexusMods.MkDocsMaterial.Themes.Next.

Reply:
You're welcome to do so.

Reply:
Well no, we want to keep the styling consistent across the projects.

Reply:
Eh, scratch that, the perms on that repo are borked and need to be updated. I'll work on that sometime in the future